### PR TITLE
Update test failure expectations for Firefox 31

### PR DESCRIPTION
**Pull request by alancutter**

### CODE REVIEW

File: test/testcases/auto-test-path.html [hunk 1]
```diff
@@ -106,7 +106,7 @@
     ],
     message: 'Doesn\'t quite follow path correctly.',
   }, {
-    browser_configurations: [{ firefox: true, version: '28|29|30' }],
+    browser_configurations: [{ firefox: true, version: '28|29|30|31' }],
```
Comment (dstoc) [inline test/testcases/auto-test-path.html]: Are there any versions of ff where these expectations aren't used? Do we still test < 28?

Comment (alancutter) [inline test/testcases/auto-test-path.html]: Removed untested Firefox versions.

**Comment by alancutter** [comment]
dstockwell: Poke.

**Comment by dstoc** [comment]
lgtm

**Pull request merged**